pkg/vm/cli/cli.go

```diff
@@ -47,63 +47,67 @@ const (
 
 var commands = []cli.Command{
 	{
-		Name:        "exit",
-		Usage:       "Exit the VM prompt",
-		Description: "Exit the VM prompt",
-		Action:      handleExit,
+		Name:   "exit",
+		Usage:  "Exit the VM prompt",
+		Action: handleExit,
 	},
 	{
-		Name:        "ip",
-		Usage:       "Show current instruction",
-		Description: "Show current instruction",
-		Action:      handleIP,
+		Name:   "ip",
+		Usage:  "Show current instruction",
+		Action: handleIP,
 	},
 	{
-		Name:      "break",
-		Usage:     "Place a breakpoint",
-		UsageText: `break <ip>`,
-		Description: `break <ip>
-<ip> is mandatory parameter, example:
-> break 12`,
+		Name:  "break",
+		Usage: "Place a breakpoint at the specified instruction",
+		Flags: []cli.Flag{
+			cli.IntFlag{
+				Name:     "ip, i",
```

Comment:
So we need to type `break --ip 12` instead of `break 12`? That's a regression, it shouldn't be like that.

Reply:
OK, reverted.